create_docs.sh

```diff
@@ -6,17 +6,17 @@ die() { echo "$*" 1>&2 ; exit 1; }
 
 workdir=$PWD
 
-# rm -rf clones # thoth
-# mkdir -p thoth/
+rm -rf clones # thoth
+mkdir -p thoth/
 for repo in storages
 do
-    # git clone --depth 1 https://github.com/thoth-station/${repo}.git clones/${repo}
+    git clone --depth 1 https://github.com/thoth-station/${repo}.git clones/${repo}
     # Copy _templates to each repo for Google analytics functionality.
     if  [[ $GITHUB_COMMIT = "1" ]]; then
         cp -r _templates/ clones/${repo}/docs/source/
     fi
     pushd clones/${repo}
-    # pipenv install
+    pipenv install
```

Comment:
no micropipenv here?

Reply:
Pipenv is already used during docs generation for creating env and installing deps. We can switch to micropipenv, but I would do it in a separate PR then.